lib/clean.js

```diff
@@ -1,19 +1,15 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var del = require('del');
+import {merge} from 'lodash';
+import del from 'del';
 
-var defaults = {
+const defaults = {
   dest: 'dist'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
 
-  gulp.task('clean', function (done) {
-    del(dest).then(function () {
-      done();
-    });
-  });
+  gulp.task('clean', done => del(dest).then(() => done()));
```

Comment:
@mrgerardorodriguez I'm not convinced that tests passing equate to this particular `gulp` task completing successfully (by invoking the `done` callback). A task that doesn't return correctly can perform OK in some cases but can cause woe when composed with other tasks.